### PR TITLE
Make copyedits to Contributing index.md

### PR DESCRIPTION
**Pull request by tiffany76**
This PR fixes a few small typos and clarity issues on the [Contributing](https://opentelemetry.io/docs/contributing/) page.

### CODE REVIEW

**Review by svrnm — APPROVED** [review]

**Comment by svrnm** [comment]
thank you @tiffany76 !

**Review by theletterf — APPROVED** [review]
Thanks!

File: content/en/docs/contributing/_index.md [hunk 1]
```diff
@@ -401,28 +401,28 @@ To build and serve the site locally with Hugo, run the following command:
...
 
 To stop the local Hugo instance, go back to the terminal and type `Ctrl+C`, or
 close the terminal window.
 
 #### Site deploys and PR previews
 
-If you submit a PR, Netlify will create a [deploy preview][] so that you can
-review your changes. Once your PR is merged, Netlify deploys the updated site to
-the production server.
+If you submit a PR, Netlify creates a [deploy preview][] so that you can review
```
Comment (theletterf) [inline content/en/docs/contributing/_index.md:412]: Love the switch to present tense here!

File: content/en/docs/contributing/_index.md [hunk 2]
```diff
@@ -523,21 +523,21 @@ create a merge conflict. You must resolve all merge conflicts in your PR.
...
+- Approved by at least one approver.
+- No failing PR checks.
+- PR branch is up-to-date with the base branch.
 
 > **Important**
 >
 > Do not worry too much about failing PR checks. Community members will help you
 > to get them fixed, by either providing you with instructions how to fix them
-> or by fixing them on your behave.
+> or by fixing them on your behalf.
```
Comment (theletterf) [inline content/en/docs/contributing/_index.md:536]: Gee, I wonder how this one got through :-S

File: content/en/docs/contributing/_index.md [hunk 3]
```diff
@@ -523,21 +523,21 @@ create a merge conflict. You must resolve all merge conflicts in your PR.
...
 
 ## Open an issue
 
-If you want to suggest improvements to existing content or notice an error, open
+If you notice an error or want to suggest improvements to existing content, open
```
Comment (theletterf) [inline content/en/docs/contributing/_index.md:540]: 👏🏻

**Pull request merged**